app/javascript/components/App.js

```diff
@@ -61,73 +57,64 @@ class App extends React.Component {
   };
 
   addToFavorites = (drink) => {
-    drink.drinkIngredients = drink.drinkIngredients.join("")
-    const {drinkIngredients, drinkName, drinkInstructions, drinkThumb, _id} = drink    
-    const favObj = { 
-      name:drinkName,
-      img_url:drinkThumb,
-      steps:drinkInstructions,
-      ingredients:drinkIngredients,
-      api_id:_id
-    }
-    this.postToFavorites(favObj)
-
-    }
-
-    // deleteDrink = (id) => {
-    //   fetch(`http://localhost:3000/drinks_list/${id}`, {
-    //   headers: {
-    //     "Content-Type": "application/json"
-    //   },
-    //   method: "DELETE"
-    // })
-    // .then(response => {
-    //   return response.json()
-    // })
-    // .then(payload => {
-    //   return this.readDrinks()
-    // })
-    // .catch(errors => {
-    //   console.log("delete errors:", errors)
-    // })
-    // }
+    drink.drinkIngredients = drink.drinkIngredients.join("");
+    const { drinkIngredients, drinkName, drinkInstructions, drinkThumb, _id } =
+      drink;
+    const favObj = {
+      name: drinkName,
+      img_url: drinkThumb,
+      steps: drinkInstructions,
+      ingredients: drinkIngredients,
+      api_id: _id,
+    };
+    this.postToFavorites(favObj);
+  };
```

Comment:
`addToFavorites` naming suggests that it might be doing the same thing as `postToFavorites()`. It looks like it's really building the request object so `addToFavorites` should be renamed to reflect that for clarity